master/usage/configuration/bgp.md

```diff
@@ -96,21 +96,57 @@ to ensure continuity of service.
 
 #### Example
 
-To turn off the full BGP node-to-node mesh, or to modify the global AS number,
-run the following command on any node:
-
-```
-# Get the current bgpconfig settings
-$ calicoctl get bgpconfig -o yaml > bgp.yaml
-
-# Modify nodeToNodeMeshEnabled to true or false
-# -or-
-# modify the asNumber to appropropriate integer
-$ vim bgp.yaml
-
-# Replace the current bgpconfig settings
-$ calicoctl replace -f bgp.yaml
-```
+**Prerequisite**: `calicoctl` [installed]({{site.baseurl}}/{{page.version}}/usage/calicoctl/install) and [configured]({{site.baseurl}}/{{page.version}}/usage/calicoctl/configure/).
+
+To turn off the full BGP node-to-node mesh or to modify the global AS number, complete
+the following steps.
+
+1. Issue the following command to determine if you have a `default` BGP configuration
+   resource.
+   
+    ```
+    calicoctl get bgpconfig default
+    ```
+
+1. If the resource _does_ exist, skip to step 3. Otherwise, use the following 
+   command to create the resource. Before issuing the command, adjust the 
+   `nodeToNodeMeshEnabled` and `asNumber` lines and values as desired. 
+   Refer to [BGP Configuration Resource]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/bgpconfig) for details about these settings.
+
+    ```
+    cat << EOF | calicoctl create -f -
+    apiVersion: projectcalico.org/v3
+    kind: BGPConfiguration
+    metadata:
+      name: default
+    spec:
+      logSeverityScreen: Info
+      nodeToNodeMeshEnabled: false
+      asNumber: 63400
+    ```
+    
+    You're done!
+
+1. If the resource _does_ exist, use the following command to retrieve it and save it 
+   to a file.
+    
+    ```
+    calicoctl get bgpconfig default -o yaml > bgp.yaml
+    ```
+
+1. Open the bgpconfig settings file in your favorite editor, modify 
```

Comment:
typo: you said `favorite editor` and then used `vim` in the example 😛 
jk

Reply:
It's true, vim is not my favorite...vi is!